src/tests/dictionaryConcepts/reducer/dictionaryConcept.test.js

```diff
@@ -27,40 +27,39 @@ import {
   REMOVE_CONCEPT,
 } from '../../../redux/actions/types';
 
-let state;
-let action;
-
-beforeEach(() => {
-  state = {
-    concepts: [],
-    loading: false,
-    dictionaryConcepts: [paginatedConcepts.concepts],
-    paginatedConcepts: [paginatedConcepts.concepts],
```

Comment:
Is this a formatting change?

Reply:
I restructured it to avoid repetition and have two tests in the same describe block